content/sensu-go/6.3/observability-pipeline/observe-schedule/metrics.md

```diff
@@ -118,285 +119,471 @@ The [example metric check][6] will produce events similar to this metric event:
+      },
+      {
+        "name": "sensu-centos.disk_usage.root.used_percentage",
+        "value": 4,
+        "timestamp": 1625000586,
+        "tags": null
+      },
+      {
+        "name": "sensu-centos.disk_usage.root.dev.used",
+        "value": 0,
+        "timestamp": 1625000586,
+        "tags": null
+      },
+      {
+        "name": "sensu-centos.disk_usage.root.dev.avail",
+        "value": 485,
+        "timestamp": 1625000586,
+        "tags": null
+      },
+      {
+        "name": "sensu-centos.disk_usage.root.dev.used_percentage",
+        "value": 0,
+        "timestamp": 1625000586,
+        "tags": null
+      },
+      {
+        "name": "sensu-centos.disk_usage.root.run.used",
+        "value": 51,
+        "timestamp": 1625000586,
+        "tags": null
+      },
+      {
+        "name": "sensu-centos.disk_usage.root.run.avail",
+        "value": 446,
+        "timestamp": 1625000586,
+        "tags": null
+      },
+      {
+        "name": "sensu-centos.disk_usage.root.run.used_percentage",
+        "value": 11,
+        "timestamp": 1625000586,
+        "tags": null
+      },
+      {
+        "name": "sensu-centos.disk_usage.root.boot.used",
+        "value": 130,
+        "timestamp": 1625000586,
+        "tags": null
+      },
+      {
+        "name": "sensu-centos.disk_usage.root.boot.avail",
+        "value": 885,
+        "timestamp": 1625000586,
+        "tags": null
+      },
+      {
+        "name": "sensu-centos.disk_usage.root.boot.used_percentage",
+        "value": 13,
+        "timestamp": 1625000586,
+        "tags": null
+      },
+      {
+        "name": "sensu-centos.disk_usage.root.home.used",
+        "value": 33,
+        "timestamp": 1625000586,
+        "tags": null
+      },
+      {
+        "name": "sensu-centos.disk_usage.root.home.avail",
+        "value": 20446,
+        "timestamp": 1625000586,
+        "tags": null
+      },
+      {
+        "name": "sensu-centos.disk_usage.root.home.used_percentage",
+        "value": 1,
+        "timestamp": 1625000586,
+        "tags": null
+      },
+      {
+        "name": "sensu-centos.disk_usage.root.vagrant.used",
+        "value": 79699,
+        "timestamp": 1625000586,
+        "tags": null
+      },
+      {
+        "name": "sensu-centos.disk_usage.root.vagrant.avail",
+        "value": 874206,
+        "timestamp": 1625000586,
+        "tags": null
+      },
+      {
+        "name": "sensu-centos.disk_usage.root.vagrant.used_percentage",
+        "value": 9,
+        "timestamp": 1625000586,
+        "tags": null
+      }
+    ]
+  },
+  "metadata": {
+    "namespace": "default"
+  },
+  "id": "7468a597-bc3c-4ea7-899c-51c4d2992689",
+  "sequence": 5,
+  "timestamp": 1625000586
 }
 {{< /code >}}
 
 {{< /language-toggle >}}
 
+{{% notice note %}}
+**NOTE**: Metrics data points are not included in events retrieved with `sensuctl event info` &mdash; these events include text output rather than a set of metrics points.
```

Comment:
```suggestion
**NOTE**: Metrics data points are not included in events retrieved with `sensuctl event info` &mdash; these events include check output text rather than a set of metrics points.
```